modules/sno-du-cluster-specific-addressing.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-cluster-specific-addressing_{context}"]
+= Cluster-specific addressing
+
+After networking has been configured, configure the specific addresses for the single node
```

Comment:
What configuration is the "after networking has been configured" referring to? The NMStateConfig is used to configure the specific ...